ftw/slider/upgrades/20170824081802_add_i_name_from_title_behavior_to_ftw_slider_pane/types/ftw.slider.Pane.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object name="ftw.slider.Pane">
+
+    <property name="behaviors" purge="False">
+        <element value="plone.app.content.interfaces.INameFromTitle"/>
```

Comment:
Yes, uniqueness is guaranteed. If you run an upgrade step, which add behavior, multiple times, the behavior is *not* added multiple times.